runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/DokkaCollectorTask.kt

```diff
@@ -13,42 +11,44 @@ open class DokkaCollectorTask : DefaultTask() {
     var modules: List<String> = emptyList()
 
     @Input
-    var outputDirectory: String = ""
+    var outputDirectory: String = defaultDokkaOutputDirectory().absolutePath
 
     private lateinit var configuration: GradleDokkaConfigurationImpl
 
+    @Input
+    val dokkaTaskNames: MutableSet<String> = mutableSetOf()
```

Comment:
Maybe `var` here would be better to allow replacing the list?

Reply:
I am not sure if this double mutability is really a good idea, especially because of the fact that we are populating this with a default for default tasks. 
Making this a `var` might mislead people to think that it is "ok" to just replace this entirely. If they really want to do something custom, they could still clear it. 

I am not entirely against your proposal, but I am just a little bit afraid that it makes thinks more complicated and error prone and that the potential convenience might not be worth it 🙄